content/4_Gold/TopoSort.mdx

```diff
@@ -451,6 +451,44 @@ public class TopoSort {
 
 </JavaSection>
 
+<PySection>
+
+```py
+from collections import deque
+
+def bfs(n, G):
```

Comment:
just put this in the main function
(unless it tle's w/o)

Reply:
also type annotations

Reply:
just put `bfs()` in main function and add type annotations

Reply:
I'm willing to use better variable names, can you give me some advice?

Reply:
Ok I just updated some.